TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/teleop/TeleOpMain.java

```diff
@@ -33,6 +34,8 @@ public void runOpMode() {
 
         telemetry = new MultipleTelemetry(telemetry, FtcDashboard.getInstance().getTelemetry());
 
+        robot.fr.setDirection(DcMotorSimple.Direction.REVERSE);
```

Comment:
Can we move this into the Robot class? This way it will apply to every program, not just TeleOpMain.

Reply:
Go for it